services/bugmon/launch-bugmon.sh

```diff
@@ -31,7 +32,7 @@ persist-limit: 0
 EOF
 
 # Copy pernosco-shared to poetry python virtual env
-BASE_PY_PATH="$(python3 -c 'import distutils.sysconfig;print(distutils.sysconfig.get_python_lib())')"
+BASE_PY_PATH="$(/opt/venvs/pernosco/bin/python3 -c 'import distutils.sysconfig;print(distutils.sysconfig.get_python_lib())')"
 POETRY_PY_PATH="$(poetry run python3 -c 'import distutils.sysconfig;print(distutils.sysconfig.get_python_lib())')"
 cp -r "$BASE_PY_PATH/pernoscoshared" "$POETRY_PY_PATH"
```

Comment:
Sure, but I think this would be solved by using subprocess to call `pernosco-submit` too....